README.md

```diff
@@ -2654,6 +2672,22 @@ An array of hashes used to override the [ErrorDocument](https://httpd.apache.org
     }
 ~~~
 
+###### `ext_filter_options`
+
+Sets the [ExtFilterOptions](http://httpd.apache.org/docs/current/mod/mod_ext_filter.html) directive.
+Note that you must delcare `class {'apache::mod::ext_filter': }` before using this directive.
```

Comment:
i'm not sure if i'm missing a bigger context here, but it's not clear from this documentation that this is only valid in directory context.